server/api/reviews_eunji.js

```diff
@@ -0,0 +1,39 @@
+const router = require('express').Router();
+const { Review } = require('../db/models');
+module.exports = router;
+
+// see all reviews
+router.get('/', (req, res, next)=>{
+  Review.findAll()
+    .then( reviews => res.json(reviews))
+    .catch(next)
+})
+// see all reviews by userId
+router.get('/users/:id', (req, res, next)=>{
+  Review.findById({ where: { userId: req.params.id }})
+    .then( reviewsByUser => res.json(reviewsByUser))
+    .catch(next)
+})
+// see all reviews by productId
+router.get('/products/:id', (req, res, next)=>{
```

Comment:
Here I wonder whether we can have a route /:productId instead of /product/:id. Same as above...